packer/run_interfaces.go

```diff
@@ -31,7 +31,7 @@ const (
 )
 
 type FixConfigOptions struct {
-	DiffOnly bool
+	Mode FixConfigMode
```

Comment:
@azr I made this change so that we could use the modes as instead. Is this what you originally intended when creating `FixConfigMode`?

Reply:
Yes ! Totally, I was about to ask you to change this on my last review but then I cancelled my message because we could have done that when we need it :D. I think doing this now is really nice because we have the error handling and etc.